test/driver.js

```diff
@@ -316,6 +316,7 @@ var rasterizeXfaLayer = (function rasterizeXfaLayerClosure() {
             xfa,
             div,
             viewport: viewport.clone({ dontFlip: true }),
+            annotationStorage,
```

Comment:
Similar to the comment in `test_manifest.json`, shouldn't this provide the "correct" intent for a print test?
Note that we're doing that for the "regular" rendering tests, when `"print": true,` is set in the manifest.